### ISSUE TITLE
A way to rate articles?

### ISSUE THREAD

**Issue by TheBeege** [TheBeege/SciNet#12]
Based on the Vox article, I think we need a way to rate the quality of articles, specifically along the lines of their scientific rigor and completeness. I don't think this is something that can be done objectively; I think it needs to be crowd-sourced. I also think that it can't be something free-form. I think we need to define different dimensions, such as reproducibility, quality of analysis, bias-checking, etc etc.

Once we have a way to rate articles, we also have a way to promote and exemplify articles of high quality. This incentivizes scientists with regards to recognition. A scientist could put on their resume or CV that they have X # of articles that are held in high regard on SciNet for study quality. Exactly how we do that belongs in another issue, however. 

So to close this issue, I'd like to first decide whether or not we want an article rating system, followed by a first-pass at what that system might look like.

**Comment by anthonyselkowitz** [comment]
I agree with the rating system and I agree with the categories of ratings. Additionally, should it be a net positive rating scale such as Reddit employs? People could vote up or down on the rating scale.  Also, I think that a feedback section and requests for feedback would be good as well. So a qualitative section so the researchers could understand why others would rate that section poorly or highly.
